plugins/outputs/application_insights/application_insights.go

```diff
@@ -283,11 +283,9 @@ func toFloat64(value interface{}) (float64, error) {
 func toInt(value interface{}) (int, error) {
 	if !is32BitChecked {
 		is32BitChecked = true
-		var i int
-		if unsafe.Sizeof(i) == 4 {
+		is32Bit = false
+		if strconv.IntSize == 32 {
 			is32Bit = true
-		} else {
-			is32Bit = false
 		}
```

Comment:
How about
```golang
		is32Bit = strconv.IntSize == 32
```

Reply:
Now that it is a simple equation, maybe the global `is32BitChecked` construct can be removed as well? It doesn't seem necessary anymore?

So you can just have this:
```go
	switch v := value.(type) {
	case uint64:
		if strconv.IntSize == 32 {
```

Reply:
@Hipska Perfect! Changed.